databases/week3/homework3.sql

```diff
@@ -0,0 +1,160 @@
+
+ --create Meal table: 
+CREATE TABLE Meal(
+ id int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,  
+ title varchar(255) NOt NULL,
+ description TEXT  NOT NULL,
+ location varchar(255) NOT NULL,
+ `when` DATETIME NOT NULL, 
```

Comment:
Why did you put when in ``?

Reply:
Hi . I used backtick with when ,  because when is a reserved key :)